src/v2/BaseLoginRouter.js

```diff
@@ -184,10 +184,11 @@ export default Router.extend({
       return this.handleUpdateAppState(error.details);
     }
 
-    // assume it's a config error
-    this.settings.callGlobalError(new Errors.ConfigError(
-      error
-    ));
+    // If the error is a string, wrap it in an Error object
+    if (typeof error === 'string') {
```

Comment:
In which case the error is coming as `string` and not as `json` object ?

Reply:
In a case where somebody messed up. All errors should be instances of an Error class (or subclass) so that they have a proper `stack`.